gar/CMakeLists.txt

```diff
@@ -22,7 +19,12 @@ target_link_libraries(aligator_gar PUBLIC proxsuite-nlp::proxsuite-nlp)
 if(BUILD_WITH_OPENMP_SUPPORT)
   target_link_libraries(aligator_gar PUBLIC OpenMP::OpenMP_CXX)
 endif()
-target_link_libraries(aligator_gar PUBLIC Tracy::TracyClient)
+# Link to tracy, and add installed tracy headers to install interface
+# for other targets to consume us without having to get tracy.
+target_link_libraries(aligator_gar PRIVATE Tracy::TracyClient)
+target_include_directories(
+  aligator_gar INTERFACE $<INSTALL_INTERFACE:${THIRD_PARTY_HEADERS_INSTALL_DIR}>
```

Comment:
Don't forget the BUILD_INTERFACE to allow workspace to work.

Reply:
It should be okay, no ? At build time the tracy headers are obtained from the submodule, and at install they're in the third-party/ dir. For boost they're always in the third-party/ dir

Reply:
Nevermind it was actually required for building with crocoddyl compatibility